blockchain/chain/src/store/chain_store.rs

```diff
@@ -47,7 +48,7 @@ const BLOCK_VAL_PREFIX: &[u8] = b"block_val/";
 // A cap on the size of the future_sink
 const SINK_CAP: usize = 200;
 
-const DEFAULT_TIPSET_CACHE_SIZE: usize = 8192;
+const DEFAULT_TIPSET_CACHE_SIZE: Option<NonZeroUsize> = NonZeroUsize::new(8192);
```

Comment:
You could write something like:

```suggestion
// XXX: Update when const `option.unwrap()` is stable.
const DEFAULT_TIPSET_CACHE_SIZE: Option<NonZeroUsize> = {
  match NonZeroUsize::new(8192) {
    Some(usize) => usize,
    None => panic!("Why God?! Why?"),
  }};
```

Reply:
Keeping what you have is fine, though.